### PR TITLE
Revert "Update links for @jhildenbiddle"

### PR DESCRIPTION
**Pull request by sy-records**

### CODE REVIEW

**Review by Koooooo-7 — CHANGES_REQUESTED** [review]
I think it's not that strict about the links. 
Check all this kind of @ links , there have both project and profile locations. not just this one is specific.

**Comment by sy-records** [comment]
[jthegedus/github-action-awesome-lint](https://github.com/jthegedus/github-action-awesome-lint) is now deprecated, it recommends using `npx -y awesome -lint 'path/to/file.md'` instead, are the two effects the same?

File: README.md [hunk 1]
```diff
@@ -156,7 +156,7 @@ A curated list of awesome things related to <a href="//docsify.js.org">docsify</
 - [docsify-demo-box-vue](https://github.com/njleonzhang/docsify-demo-box-vue) - Write Vue demo in docsify with instant preview and jsfiddle integration [@njleonzhang](https://github.com/njleonzhang/).
 - [docsify-demo-box-react](https://github.com/njleonzhang/docsify-demo-box-react) - Write React jsx demo in docsify with instant preview and jsfiddle integration @njleonzhang.
 - [docsify-edit-on-github](https://github.com/njleonzhang/docsify-edit-on-github) - Add Edit on GitHub button on every pages @njleonzhang.
-- [docsify-tabs](https://jhildenbiddle.github.io/docsify-tabs) - A docsify plugin for displaying tabbed content from markdown [@jhildenbiddle](https://github.com/jhildenbiddle/).
+- [docsify-tabs](https://jhildenbiddle.github.io/docsify-tabs) - A docsify plugin for displaying tabbed content from markdown [@jhildenbiddle](https://github.com/jhildenbiddle/docsify-tabs).
```
Comment (Koooooo-7) [inline README.md:159]: This located the author name to the project.

File: README.md [hunk 2]
```diff
@@ -156,7 +156,7 @@ A curated list of awesome things related to <a href="//docsify.js.org">docsify</
...
 - [docsify-katex](https://github.com/upupming/docsify-katex) - A docsify plugin for rendering LaTex math equations [@upupming](https://github.com/upupming).
```
Comment (Koooooo-7) [inline README.md:160]: This author name is located to his profile.

**Review by Koooooo-7 — COMMENTED** [review]
Why they are different ?

**Pull request closed without merging**